recipes/nsimd/2.x/test_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class NsimdTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
```

Comment:
We had a new rule come out recently #7153 

```suggestion
    generators = "cmake", "cmake_find_package_multi"
```

You'll also need to call `find_package` in the cmake for the test package :)